haskell-tree-sitter.cabal

```diff
@@ -26,21 +26,14 @@ library
                      , TreeSitter.Node
                      , TreeSitter.Tree
                      , TreeSitter.Ptr
-  Include-dirs:        includes                     
-  Includes:            tree_sitter_ptr.h                     
+  Include-dirs:        includes
+                     , vendor/tree-sitter/lib/include
+                     , vendor/tree-sitter/lib/src
+                     , vendor/tree-sitter/lib/utf8proc
+  Includes:            tree_sitter_ptr.h
   c-sources:           src/bridge.c
                      , src/bridge_ptr.c
-                     , vendor/tree-sitter/externals/utf8proc/utf8proc.c
-                     , vendor/tree-sitter/src/runtime/get_changed_ranges.c
-                     , vendor/tree-sitter/src/runtime/language.c
-                     , vendor/tree-sitter/src/runtime/lexer.c
-                     , vendor/tree-sitter/src/runtime/node.c
-                     , vendor/tree-sitter/src/runtime/parser.c
-                     , vendor/tree-sitter/src/runtime/stack.c
-                     , vendor/tree-sitter/src/runtime/subtree.c
-                     , vendor/tree-sitter/src/runtime/tree.c
-                     , vendor/tree-sitter/src/runtime/tree_cursor.c
-                     , vendor/tree-sitter/src/runtime/utf16.c
+                     , vendor/tree-sitter/lib/src/lib.c
```

Comment:
There's now a single 'amalgamation build' file, `lib.c`, so we don't have to list out all of the source files here.